plugins/lookup/vmware_moid.py

```diff
@@ -0,0 +1,526 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+lookup: vmware_moid
+short_description: Look up MOID for vSphere objects using vCenter REST API.
+description:
+    - Returns MOID of the object contained in the specified path.
+options:
+    _terms:
```

Comment:
Instead of `_terms`, use individual parameters like `object_type='vm'`, `path=''`

Reply:
Not yet addressed.